content/docs/command-reference/commit.md

```diff
@@ -62,26 +64,26 @@ There are many cases where the last step is not desirable (for example rapid
 iterations on an experiment). The `--no-commit` option prevents the last step
 from occurring (on the commands where it's available), saving time and space by
 not storing unwanted <abbr>data artifacts</abbr>. The file hash is still
-computed and added to the DVC-file, but the actual data file is not saved in the
-cache. This is where the `dvc commit` command comes into play. It performs that
-last step (saving the data in cache).
+computed and added to the `dvc.lock` or `.dvc` file, but the actual data file is
+not saved in the cache. This is where the `dvc commit` command comes into play.
+It performs that last step (saving the data in cache).
 
 Note that it's best to avoid the last two scenarios. They essentially
-force-update the [DVC-files](/doc/user-guide/dvc-files-and-directories) and save
-data to cache. They are still useful, but keep in mind that DVC can't guarantee
-reproducibility in those cases.
+force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
+still useful, but keep in mind that DVC can't guarantee reproducibility in those
+cases.
 
 ## Options
 
 - `-d`, `--with-deps` - determines files to commit by tracking dependencies to
-  the target DVC-files (stages). If no `targets` are provided, this option is
-  ignored. By traversing all stage dependencies, DVC searches backward from the
-  target stages in the corresponding pipelines. This means DVC will not commit
-  files referenced in later stages than the `targets`.
+  the target stages or `.dvc` files. If no `targets` are provided, this option
+  is ignored. By traversing all stage dependencies, DVC searches backward from
+  the target stages in the corresponding pipelines. This means DVC will not
+  commit files referenced in later stages than the `targets`.
 
 - `-R`, `--recursive` - determines the files to commit by searching each target
-  directory and its subdirectories for DVC-files to inspect. If there are no
-  directories among the `targets`, this option is ignored.
+  directory and its subdirectories for stages or `.dvc` files to inspect. If
+  there are no directories among the `targets`, this option is ignored.
```

Comment:
- [ ] Should probably use ``stages (in `dvc.yaml`) or `.dvc` files``

But let's leave it for another PR since this option is also in several refs.